constants/index.ts

```diff
@@ -246,8 +246,17 @@ export const PAYBUTTON_TRANSACTIONS_FILE_HEADERS = {
   address: 'Address'
 }
 
+export const PAYBUTTON_TRANSACTIONS_FILE_HEADERS = { ...BASE_FILE_HEADERS }
+
+export const PAYBUTTON_PAYMENT_FILE_HEADERS = {
+  ...BASE_FILE_HEADERS,
+  collapsed: 'Collapsed',
+  notes: 'Notes'
+}
+
 export const DEFAULT_PAYBUTTON_CSV_FILE_DELIMITER = ','
 export const MAX_RECORDS_PER_FILE = 10000
+export const DEFAULT_COLLAPSE_THRESHOLD_FILE_USD = 1
```

Comment:
How about changing this from USD to `DEFAULT_CSV_COLLAPSE_THRESHOLD` and in a follow-up branch, we can make this configurable in the account settings? The threshold will use whatever currency they have set, not just USD.